_posts/2018-12-17-Devoxx-MA.md

```diff
@@ -0,0 +1,245 @@
+Yannick shared his experiences and mentioned that even though there is an [incubating, opt-in feature](https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods){:target="_blank" rel="noopener noreferrer"} to allow Mockito to mock classes and functions, the user experience wasn't all that nice.
+
+He looked for other frameworks and stumbled upon [MockK](https://mockk.io){:target="_blank" rel="noopener noreferrer"}, a mocking framework specifically created for Kotlin by [Oleksiy Pylypenko](https://github.com/oleksiyp){:target="_blank" rel="noopener noreferrer"}.
+MockK’s main philosophy is offering first-class support for Kotlin features and being able to write idiomatic Kotlin code when using it.
+He was pretty enthusiastic about MockK and went over all its features and its ease of use. 
+
+There is also a [blog post](/testing/2018/02/05/Writing-tests-in-Kotlin-with-MockK.html){:target="_blank" rel="noopener noreferrer"} written by Yannick specifically about his experiences with Mockito and MockK in Kotlin projects.
+
+<script async class="speakerdeck-embed" data-id="c67c9a2029ad4b0e9c0016b55bb68d10" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
+<br/>
+
+## Stream Processing Live Traffic Data with Kafka Streams by [Tom Van den Bulck](https://twitter.com/tomvdbulck){:target="_blank" rel="noopener noreferrer"} and [Tim Ysewyn](https://twitter.com/TYsewyn){:target="_blank" rel="noopener noreferrer"}
+<span class="image left"><img class="p-image" alt="Tom van den Bulck" src="/img/devoxx-ma-2018/speaker-tom-van-den-bulck.png"></span>
+<span class="image left"><img class="p-image" alt="Tim Ysewyn" src="/img/devoxx-ma-2018/speaker-tim-ysewyn.png"></span>
+
+Our colleagues, Tim Ysewyn and Tom Van den Bulck, gave a talk about stream processing live traffic data with Kafka Streams.
+
+Tom did the theoretical part of the talk starting with the bigger picture. 
+He explained the stream processing concept which is basically computing data directly as it is produced or received.
+
+<img class="image fit" src="{{ '/img/devoxx-ma-2018/01_data_at_rest_infrastructure.png' | prepend: site.baseurl }}" alt="Data at rest infrastructure" width="60%" />
+
+In the image above we can see an example without stream processing.
+In this case, the data is stored in databases and file storages.
+Using a scheduler, applications can retrieve and process the stored data.
+
+<img class="image fit" src="{{ '/img/devoxx-ma-2018/02_stream_processing_infrastructure.png' | prepend: site.baseurl }}" alt="Stream processing infrastructure" width="60%" />
+
+With stream processing the data will be processed directly as streams of events, recreating other event streams for other applications if needed.
+The applications will react on an event instead of scheduling jobs to retrieve and process data stored in databases and file storages.
```

Comment:
```suggestion
The applications will react on events instead of scheduling jobs to retrieve and process data stored in databases and file storages.
```